src/docs/project_goals.md

```diff
@@ -0,0 +1,48 @@
+
+## VS Code compatibility
+
+Theia provides the ability to host VS Code extensions. This allows adopters to benefit from features that are provided as VS Code extensions and make them part of their tool offering. It also allows end users of Theia-based tools to install additional features, if the adopter providing a tool allows users to do so. When applicable, the default Theia UX aligns with the VS Code UI, although adopters can deviate from this.
+
+## Use standards and don’t reinvent the wheel
+
+Theia uses/reuses industrial standard technologies and practices whenever applicable. This keeps the scope of the project minimal and decreases the maintenance cost. This applies to the use of frameworks and development tools as well as to general concepts such as UX.
+
+## Product Templates
+
+The Eclipse Theia project does not primarily aim to provide products for end users but focuses on offering a platform for building products. However, the project provides product templates, a.k.a. “Blueprints”. These blueprints serve two purposes. First, they allow adopters to consume example Theia-based products from the view point of an end user, which enables them to evaluate the underlying platform without first creating a product based on it. Second, the blueprints serve as templates to create custom products. Therefore, the template products contain documentation on how to customize them.
+
+## Open and Vendor Neutral Governance
+
+Eclipse Theia is an open and vendor-neutral project with a diverse community of contributors from large corporations down to small companies and even individual developers . This diverse and steadily growing community is a stable base for the ongoing development of additional features and maintenance of existing functionality based on a broad and balanced view of the requirements put forward by the Theia adopters. The commercial-friendly licensing and the rigorous underlying IP management ensures that adopters can safely build commercial and internal products based on Theia. Project communication is open and transparent and welcomes new adopters by keeping the barriers of contribution as low as possible. Decisions in the project are based on the principle of meritocracy meaning that the weight of contributors in decisions is solely based on their achievements for the project as recognized by their peers. This provides an important incentive for contributors to make contributions. The open governance and diverse community of Theia is one of its strongest assets for its mid- to long-term evolution and viability.
```

Comment:
minor: extra whitespace before the period:

`even individual developers .` > `even individual developers.`


```suggestion
Eclipse Theia is an open and vendor-neutral project with a diverse community of contributors from large corporations down to small companies and even individual developers. This diverse and steadily growing community is a stable base for the ongoing development of additional features and maintenance of existing functionality based on a broad and balanced view of the requirements put forward by the Theia adopters. The commercial-friendly licensing and the rigorous underlying IP management ensures that adopters can safely build commercial and internal products based on Theia. Project communication is open and transparent and welcomes new adopters by keeping the barriers of contribution as low as possible. Decisions in the project are based on the principle of meritocracy meaning that the weight of contributors in decisions is solely based on their achievements for the project as recognized by their peers. This provides an important incentive for contributors to make contributions. The open governance and diverse community of Theia is one of its strongest assets for its mid- to long-term evolution and viability.
```